src/ImgixTransformer.js

```diff
@@ -1,7 +1,10 @@
 import ImgixClient from './ImgixClient'
 
 const absolutePathReg = /^https?/
-const imgElementSearchReg = /(<img[^>]+>)/gi
+const imgElementReg = /(<img[^>]+>)/gi
+const imgSrcReg = /src="([^"]+)"/i
```

Comment:
this is rather a general src regex than a particular `imgSrcReg`, so it could be named `srcReg` or `srcAttrReg`.